engine/src/main/java/org/terasology/rendering/dag/nodes/AmbientOcclusionPassesNode.java

```diff
@@ -13,13 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.terasology.rendering.dag;
```

Comment:
On your first paragraph: sure. Let's add 

`//TODO: consider removing the word "Node" from the name of all Node implementations now that they are in the dag.nodes package. `

to the Node interface.

On your second paragraph: indeed it doesn't make sense to go through individual commits unless the individual commits have been "designed" and documented to be reviewed separately or skipped altogether. =)